simulationTool/components/HelpSection/TutorialCreateEnsemble.vue

```diff
@@ -0,0 +1,78 @@
+<script>
+import TutorialCreateEnsemble1 from '../../resources/img/TutorialCreateEnsemble1.png'
+import TutorialCreateEnsemble2 from '../../resources/img/TutorialCreateEnsemble2.png'
+import TutorialCreateEnsemble3 from '../../resources/img/TutorialCreateEnsemble3.png'
+
+export default {
+    name: "TutorialCreateEnsemble",
+    data() {
+        return {
+            TutorialCreateEnsemble1,
+            TutorialCreateEnsemble2,
+            TutorialCreateEnsemble3
+        }
+    }
+};
+</script>
+
+<template>
+    <div class="tutorial-page">
```

Comment:
The `TutorialPage` should become a separate component which could be reused in very `Tutorial*.vue` file

You could pass in an array of questions and answers and handle these dyanmically.
This would avoid repeated code (most important <style>) in every `Tutorial*.vue` file.

Reply:
That's a very good idea! I just followed your suggestion.